.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "gitversion.tool": {
+      "version": "5.12.0",
```

Comment:
Cake installs its own tools in a separate folder. This gitversion is for being directly called from the script steps in TC.